src/Message/Message.tsx

```diff
@@ -10,17 +10,24 @@ export type MessageProps = {
   type?: 'info' | 'warning' | 'warning-bubble'
   alignIcon?: 'center' | 'flex-start' | 'flex-end'
   backgroundColor?: string
+  sizeSmall?: boolean
 }
 
 export const Message: FC<MessageProps> = ({
   children,
   type = 'info',
   alignIcon = 'center',
   backgroundColor,
+  sizeSmall,
 }) => (
-  <Wrapper type={type} backgroundColor={backgroundColor}>
+  <Wrapper type={type} backgroundColor={backgroundColor} sizeSmall={sizeSmall}>
+    {sizeSmall}
```

Comment:
this looks odd placed here

Reply:
Definite error not sure how that got there lol, will remove